Duplicati/Library/Backend/OAuthHelper/JSONWebHelper.cs

```diff
@@ -420,7 +455,17 @@ public async Task<HttpWebResponse> GetResponseWithoutExceptionAsync(AsyncHttpReq
                 if (wex.Response is HttpWebResponse response)
                     return response;
 
-                throw;
+                /*
+                 * If not an HttpWebResponse then grab the error detail from the response stream
+                 */
+                using (Stream exstream = wex.Response.GetResponseStream())
+                {
+                    using (StreamReader reader = new StreamReader(exstream))

```

Comment:
I don't think you can guarantee there to be a response stream present? If the error is "Network disconnected" there will be no response stream and the user will get an exception here.